module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -658,39 +673,22 @@ public function getHolding($bibId, array $patron = null, array $options = [])
      */
     protected function getDueDate($itemId, $showTime)
     {
-        $dueDateValue = '';
```

Comment:
Removed the $dueDateValue string, since we can just return directly when we find a relevant value.